### PR TITLE
done: Set race starttime

### PR DESCRIPTION
**Pull request by cz8s**
we need the starttime before we can calculate the results.
Thats why #74 will depend on this.
But this is not ready yet. Somehow I don't understand why
race.hasStarted() returns false.
Will see tomorrow

### CODE REVIEW

**Comment by cz8s** [comment]
okay. everything works now. can be merged

**Comment by lplotni** [comment]
SnapCI pipeline config. for PR was out-of-date, therefore it was still using postgres9.4 and not 9.5

File: views/admin/after.jade [hunk 1]
```diff
@@ -0,0 +1,17 @@
+extends ../layout
+
+block content
+
+  h3 Startzeit
+
+  form#payment-details(method='POST', action='after')
```
Comment (lplotni) [inline views/admin/after.jade]: the ``payment-details`` ID is a C&P artefact, isn't it?

Comment (cz8s) [inline views/admin/after.jade]: yes. I fixed it in the functional tests that are not yet in a pr because they don't work...
will fix this. one sec

**Pull request merged**